<ui version="4.0" >
 <class>RapidTools</class>
 <widget class="QWidget" name="RapidTools" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>453</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <widget class="QTabWidget" name="tabWidget" >
     <property name="tabPosition" >
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <attribute name="title" >
       <string>Verify RS.com links</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Links:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QTextEdit" name="textLinks" >
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="pushCheck" >
         <property name="text" >
          <string>Verify links</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QPushButton" name="pushDownload" >
         <property name="text" >
          <string>Download working links</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>181</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <attribute name="title" >
       <string>Decode RapidSafe links</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" colspan="2" >
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Enter RapidSafe/RapidSave links:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3" >
        <widget class="QTextEdit" name="textLinksSrc" >
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Decoded links:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3" >
        <widget class="QTextEdit" name="textLinksDst" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0" >
        <widget class="QPushButton" name="pushDecode" >
         <property name="text" >
          <string>Decode</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" >
        <widget class="QPushButton" name="pushDownload2" >
         <property name="text" >
          <string>Download decoded</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>251</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3" >
      <attribute name="title" >
       <string>Extract files from a RS.com folder</string>
      </attribute>
      <layout class="QGridLayout" >
       <item row="0" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>RS.com folder URL:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="2" >
        <widget class="QLineEdit" name="lineURL" />
       </item>
       <item row="0" column="3" >
        <widget class="QPushButton" name="pushExtract" >
         <property name="text" >
          <string>Extract</string>
         </property>
         <property name="autoDefault" >
          <bool>true</bool>
         </property>
         <property name="default" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="4" >
        <widget class="QTextEdit" name="textFolderContents" >
         <property name="readOnly" >
          <bool>true</bool>
         </property>
         <property name="acceptRichText" >
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2" >
        <widget class="QPushButton" name="pushDownloadFolder" >
         <property name="text" >
          <string>Download extracted</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <spacer>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" >
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineURL</sender>
   <signal>returnPressed()</signal>
   <receiver>pushExtract</receiver>
   <slot>animateClick()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>221</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel" >
     <x>477</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
